src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -792,6 +792,70 @@ def columns(self) -> list[str]:
         """List of column names."""
         return self.frame.columns
 
+    def map_to_aois(
+        self,
+        aoi_dataframe: pm.stimulus.TextStimulus,
+        *,
+        eye: str = 'auto',
+    ) -> None:
+        """Map gaze data to aois.
+
+        Parameters
+        ----------
+        aoi_dataframe: pm.stimulus.TextStimulus
+            Area of interest dataframe.
+        eye: str
+            String specificer for inferring eye components. Supported values are: auto, mono, left
+            right, cyclops. Default: auto.
+        """
+        if 'pixel' in self.columns:
+            if eye == 'left':
+                x_eye = 'pixel_xl'
+                y_eye = 'pixel_yl'
+            elif eye == 'right':
+                x_eye = 'pixel_xr'
+                y_eye = 'pixel_yr'
+            else:
+                # auto to right eye
+                x_eye = 'pixel_xr'
+                y_eye = 'pixel_yr'
```

Comment:
here we should allow more namings as in PR #719 

example:
# suffixes as ordered after using GazeDataFrame.unnest() 
 component_suffixes = ['x', 'y', 'xl', 'yl', 'xr', 'yr', 'xa', 'ya'] 
  
 pixel_columns = ['pixel_' + suffix for suffix in component_suffixes] 
 pixel_columns = [c for c in pixel_columns if c in gaze_df.frame.columns]